parachains/runtimes/assets/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -524,3 +552,102 @@ fn foreign_pallet_has_correct_local_account() {
 	let address = Ss58Codec::to_ss58check_with_version(&account, polkadot);
 	assert_eq!(address, "13w7NdvSR1Af8xsQTArDtZmVvjE8XhWNdL4yed3iFHrUNCnS");
 }
+
+/// All configuration related to bridging
+pub mod bridging {
+	use super::*;
+	use assets_common::{matching, matching::*};
+	use sp_std::collections::btree_set::BTreeSet;
+	use xcm_builder::{NetworkExportTable, UnpaidRemoteExporter};
+
+	parameter_types! {
+		pub BridgeHubPolkadotParaId: u32 = 1002;
+		pub BridgeHubPolkadot: MultiLocation = MultiLocation::new(1, X1(Parachain(BridgeHubPolkadotParaId::get())));
+		pub BridgeHubPolkadotWithBridgeHubKusamaInstance: MultiLocation = MultiLocation::new(1, X2(Parachain(BridgeHubPolkadotParaId::get()), PalletInstance(53)));
```

Comment:
Same as for Kusama

Reply:
same as https://github.com/paritytech/cumulus/pull/2762#discussion_r1275937958